module,execution_time (sec),success,timestamp,error,error_short
cowidev.vax.batch.argentina,16.56,True,2023-06-04T07:00:23,,
cowidev.vax.batch.australia,,,2023-06-04T07:00:06,,
cowidev.vax.batch.austria_old,,,2023-06-04T07:00:06,,
cowidev.vax.batch.belgium,5.62,True,2023-06-04T07:00:12,,
cowidev.vax.batch.bolivia,3.92,True,2023-06-04T07:00:10,,
cowidev.vax.batch.canada,5.25,False,2023-06-04T07:01:56,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/canada.py"", line 282, in main
    Canada().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/canada.py"", line 270, in export
    df = df.pipe(self.pipeline, df_man)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/canada.py"", line 250, in pipeline
    .pipe(self.pipe_make_monotonic)[
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/canada.py"", line 230, in pipe_make_monotonic
    df = df.pipe(self.make_monotonic, max_removed_rows=self.max_removed_rows)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/utils/base.py"", line 120, in make_monotonic
    return mkm(
  File ""/home/owid/covid-19-data/scripts/src/cowidev/utils/utils.py"", line 23, in make_monotonic
    return make_monotonic_new(df, column_date, column_metrics, max_removed_rows)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/utils/utils.py"", line 96, in make_monotonic_new
    raise Exception(
Exception: 1 chunks of lengths 36 would be NaNed for metric people_vaccinated. That is more than maximum allowed (22) by make_monotonic() - check the data. Check dates ['2023-01-26']. Location(s) affected: {'Canada'}
",1 chunks of lengths 36 would be NaNed for metric people_vaccinated. That is more than maximum allowed (22) by make_monotonic() - check the data. Check dates ['2023-01-26']. Location(s) affected: {'Canada'}
cowidev.vax.batch.chile,1.08,True,2023-06-04T07:00:07,,
cowidev.vax.batch.czechia,37.19,True,2023-06-04T07:00:43,,
cowidev.vax.batch.denmark,,,2023-06-04T07:00:06,,
cowidev.vax.batch.ecdc,88.48,True,2023-06-04T07:01:35,,
cowidev.vax.batch.ecuador,,,2023-06-04T07:00:06,,
cowidev.vax.batch.estonia,1.13,True,2023-06-04T07:00:07,,
cowidev.vax.batch.france,4.78,True,2023-06-04T07:00:11,,
cowidev.vax.batch.germany,0.19,True,2023-06-04T07:00:06,,
cowidev.vax.batch.greece,6.96,True,2023-06-04T07:00:13,,
cowidev.vax.batch.hong_kong,10.85,True,2023-06-04T07:00:17,,
cowidev.vax.batch.indonesia,,,2023-06-04T07:00:06,,
cowidev.vax.batch.ireland,0.45,False,2023-06-04T07:01:56,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/ireland.py"", line 135, in main
    Ireland().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/ireland.py"", line 130, in export
    df = self.read().pipe(self.pipeline)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/ireland.py"", line 27, in read
    return pd.merge(data_primary, data_booster, how=""outer"", on=""date"", validate=""one_to_one"")
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/reshape/merge.py"", line 106, in merge
    op = _MergeOperation(
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/reshape/merge.py"", line 709, in __init__
    self._validate(validate)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/reshape/merge.py"", line 1429, in _validate
    raise MergeError(
pandas.errors.MergeError: Merge keys are not unique in right dataset; not a one-to-one merge
",Merge keys are not unique in right dataset; not a one-to-one merge
cowidev.vax.batch.israel,4.44,True,2023-06-04T07:00:11,,
cowidev.vax.batch.italy,20.81,True,2023-06-04T07:00:27,,
cowidev.vax.batch.japan,9.36,False,2023-06-04T07:02:05,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 375, in main
    Japan().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 359, in export
    df = self.read().pipe(self.pipeline_base)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 71, in read
    df_latest = self.read_latest().pipe(self.pipe_read_latest)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 100, in read_latest
    self._read_latest_booster(),
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 211, in _read_latest_booster
    df = self._read_xlsx(
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/japan.py"", line 233, in _read_xlsx
    raise ValueError(f""Unknown sheets: {sheets_unknown}"")
ValueError: Unknown sheets: {'5.8以降接種_総接種回数', '６回目_総接種回数', '5.8以降接種_一般接種（６回目）', '5.8以降接種_一般接種（３回目）', '5.8以降接種_一般接種（４回目）', '５回目_総接種回数', '６回目_一般接種回数', '５回目_一般接種回数', '5.8以降接種_一般接種（５回目）'}
","Unknown sheets: {'5.8以降接種_総接種回数', '６回目_総接種回数', '5.8以降接種_一般接種（６回目）', '5.8以降接種_一般接種（３回目）', '5.8以降接種_一般接種（４回目）', '５回目_総接種回数', '６回目_一般接種回数', '５回目_一般接種回数', '5.8以降接種_一般接種（５回目）'}"
cowidev.vax.batch.jersey,4.11,True,2023-06-04T07:00:10,,
cowidev.vax.batch.latvia,53.14,True,2023-06-04T07:00:59,,
cowidev.vax.batch.lithuania,4.48,True,2023-06-04T07:00:11,,
cowidev.vax.batch.luxembourg,1.73,True,2023-06-04T07:00:08,,
cowidev.vax.batch.malaysia,0.79,True,2023-06-04T07:00:07,,
cowidev.vax.batch.malta,1.04,True,2023-06-04T07:00:07,,
cowidev.vax.batch.netherlands,,,2023-06-04T07:00:06,,
cowidev.vax.batch.new_zealand,12.58,True,2023-06-04T07:00:19,,
cowidev.vax.batch.norway,0.48,True,2023-06-04T07:00:07,,
cowidev.vax.batch.peru,4.06,True,2023-06-04T07:00:10,,
cowidev.vax.batch.portugal,,,2023-06-04T07:00:06,,
cowidev.vax.batch.romania,30.97,True,2023-06-04T07:00:37,,
cowidev.vax.batch.saudi_arabia,5.89,True,2023-06-04T07:00:12,,
cowidev.vax.batch.singapore,3.98,True,2023-06-04T07:00:10,,
cowidev.vax.batch.slovakia,45.19,True,2023-06-04T07:00:51,,
cowidev.vax.batch.slovenia,3.69,True,2023-06-04T07:00:10,,
cowidev.vax.batch.south_korea,56.66,False,2023-06-04T07:03:02,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/south_korea.py"", line 209, in main
    SouthKorea().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/south_korea.py"", line 191, in export
    df_base = self.read().pipe(self.pipeline_base)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/south_korea.py"", line 43, in read
    dfs = pd.read_html(to_proxy_url(self.source_url), encoding=""utf-8"")
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/util/_decorators.py"", line 311, in wrapper
    return func(*args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/io/html.py"", line 1098, in read_html
    return _parse(
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/io/html.py"", line 906, in _parse
    tables = p.parse_tables()
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/io/html.py"", line 222, in parse_tables
    tables = self._parse_tables(self._build_doc(), self.match, self.attrs)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/io/html.py"", line 745, in _build_doc
    raise e
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/io/html.py"", line 726, in _build_doc
    with urlopen(self.io) as f:
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/io/common.py"", line 212, in urlopen
    return urllib.request.urlopen(*args, **kwargs)
  File ""/usr/lib/python3.10/urllib/request.py"", line 216, in urlopen
    return opener.open(url, data, timeout)
  File ""/usr/lib/python3.10/urllib/request.py"", line 525, in open
    response = meth(req, response)
  File ""/usr/lib/python3.10/urllib/request.py"", line 634, in http_response
    response = self.parent.error(
  File ""/usr/lib/python3.10/urllib/request.py"", line 563, in error
    return self._call_chain(*args)
  File ""/usr/lib/python3.10/urllib/request.py"", line 496, in _call_chain
    result = func(*args)
  File ""/usr/lib/python3.10/urllib/request.py"", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 500: Internal Server Error
",HTTP Error 500: Internal Server Error
cowidev.vax.batch.spc,29.53,True,2023-06-04T07:00:36,,
cowidev.vax.batch.sweden,14.98,True,2023-06-04T07:00:22,,
cowidev.vax.batch.switzerland,56.6,True,2023-06-04T07:01:04,,
cowidev.vax.batch.trinidad_and_tobago,2.81,True,2023-06-04T07:00:10,,
cowidev.vax.batch.ukraine,,,2023-06-04T07:00:07,,
cowidev.vax.batch.united_kingdom,16.11,True,2023-06-04T07:00:23,,
cowidev.vax.batch.united_states,11.01,True,2023-06-04T07:00:18,,
cowidev.vax.batch.uruguay,3.6,True,2023-06-04T07:00:12,,
cowidev.vax.batch.zimbabwe,1.01,True,2023-06-04T07:00:11,,
cowidev.vax.incremental.africacdc,,,2023-06-04T07:00:10,,
cowidev.vax.incremental.antigua_barbuda,,,2023-06-04T07:00:10,,
cowidev.vax.incremental.aruba,1.04,True,2023-06-04T07:00:11,,
cowidev.vax.incremental.austria,,,2023-06-04T07:00:10,,
cowidev.vax.incremental.azerbaijan,,,2023-06-04T07:00:10,,
cowidev.vax.incremental.bahrain,,,2023-06-04T07:00:10,,
cowidev.vax.incremental.bangladesh,42.32,True,2023-06-04T07:00:52,,
cowidev.vax.incremental.barbados,,,2023-06-04T07:00:10,,
cowidev.vax.incremental.brazil,0.51,True,2023-06-04T07:00:11,,
cowidev.vax.incremental.bulgaria,2.3,True,2023-06-04T07:00:13,,
cowidev.vax.incremental.china,20.27,True,2023-06-04T07:00:31,,
cowidev.vax.incremental.costa_rica,,,2023-06-04T07:00:10,,
cowidev.vax.incremental.croatia,0.5,True,2023-06-04T07:00:11,,
cowidev.vax.incremental.cuba,5.98,True,2023-06-04T07:00:17,,
cowidev.vax.incremental.curacao,,,2023-06-04T07:00:11,,
cowidev.vax.incremental.cyprus,0.97,True,2023-06-04T07:00:12,,
cowidev.vax.incremental.dominican_republic,13.03,False,2023-06-04T07:03:15,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/dominican_republic.py"", line 112, in main
    DominicanRepublic().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/dominican_republic.py"", line 68, in export
    data = self.read().pipe(self.pipeline)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/dominican_republic.py"", line 26, in read
    metrics = self._parse_metrics(driver)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/dominican_republic.py"", line 35, in _parse_metrics
    metric_candidates = _find_potential_metrics(driver)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/dominican_republic.py"", line 91, in _find_potential_metrics
    h3 = _find_h3(driver)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/dominican_republic.py"", line 81, in _find_h3
    raise ValueError(""More than one element found!"")
ValueError: More than one element found!
",More than one element found!
cowidev.vax.incremental.el_salvador,9.07,True,2023-06-04T07:00:20,,
cowidev.vax.incremental.faeroe_islands,0.9,True,2023-06-04T07:00:12,,
cowidev.vax.incremental.fiji,0.0,True,2023-06-04T07:00:11,,
cowidev.vax.incremental.finland,0.44,True,2023-06-04T07:00:11,,
cowidev.vax.incremental.gabon,,,2023-06-04T07:00:11,,
cowidev.vax.incremental.georgia,1.27,True,2023-06-04T07:00:12,,
cowidev.vax.incremental.greenland,,,2023-06-04T07:00:11,,
cowidev.vax.incremental.guatemala,,,2023-06-04T07:00:11,,
cowidev.vax.incremental.guernsey,,,2023-06-04T07:00:11,,
cowidev.vax.incremental.hungary,76.73,True,2023-06-04T07:01:28,,
cowidev.vax.incremental.iceland,0.88,True,2023-06-04T07:00:12,,
cowidev.vax.incremental.india,1.49,True,2023-06-04T07:00:13,,
cowidev.vax.incremental.iran,,,2023-06-04T07:00:12,,
cowidev.vax.incremental.isle_of_man,3.2,True,2023-06-04T07:00:15,,
cowidev.vax.incremental.jamaica,12.7,True,2023-06-04T07:00:25,,
cowidev.vax.incremental.kazakhstan,11.01,True,2023-06-04T07:00:23,,
cowidev.vax.incremental.kosovo,,,2023-06-04T07:00:12,,
cowidev.vax.incremental.kyrgyzstan,1.63,True,2023-06-04T07:00:14,,
cowidev.vax.incremental.laos,1.7,True,2023-06-04T07:03:17,,
cowidev.vax.incremental.macao,7.04,False,2023-06-04T07:03:24,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/macao.py"", line 90, in main
    Macao().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/macao.py"", line 75, in export
    data = self.read()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/macao.py"", line 23, in read
    iframe_url = self._get_iframe_url(driver)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/macao.py"", line 31, in _get_iframe_url
    elem = driver.find_element_by_id(""ICovid19Monitor"")
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py"", line 360, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py"", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py"", line 321, in execute
    self.error_handler.check_response(response)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py"", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {""method"":""css selector"",""selector"":""[id=""ICovid19Monitor""]""}
  (Session info: headless chrome=113.0.5672.126)

","Message: no such element: Unable to locate element: {""method"":""css selector"",""selector"":""[id=""ICovid19Monitor""]""}
  (Session info: headless chrome=113.0.5672.126)
"
cowidev.vax.incremental.moldova,,,2023-06-04T07:00:13,,
cowidev.vax.incremental.monaco,94.26,True,2023-06-04T07:01:47,,
cowidev.vax.incremental.mongolia,1.12,True,2023-06-04T07:00:14,,
cowidev.vax.incremental.montenegro,0.64,True,2023-06-04T07:00:14,,
cowidev.vax.incremental.morocco,,,2023-06-04T07:00:14,,
cowidev.vax.incremental.myanmar,,,2023-06-04T07:00:14,,
cowidev.vax.incremental.nepal,,,2023-06-04T07:00:14,,
cowidev.vax.incremental.north_macedonia,,,2023-06-04T07:00:14,,
cowidev.vax.incremental.northern_cyprus,8.55,True,2023-06-04T07:00:23,,
cowidev.vax.incremental.paho,72.06,False,2023-06-04T07:04:36,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/paho.py"", line 195, in main
    PAHO().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/paho.py"", line 190, in export
    df = self.read().pipe(self.pipeline)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/paho.py"", line 37, in read
    df = self._parse_data(url)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/paho.py"", line 68, in _parse_data
    date = self._parse_date(driver)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/paho.py"", line 96, in _parse_date
    driver.find_element_by_id(""tabZoneId77"").click()  # 87
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py"", line 360, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py"", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py"", line 321, in execute
    self.error_handler.check_response(response)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py"", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {""method"":""css selector"",""selector"":""[id=""tabZoneId77""]""}
  (Session info: headless chrome=113.0.5672.126)

","Message: no such element: Unable to locate element: {""method"":""css selector"",""selector"":""[id=""tabZoneId77""]""}
  (Session info: headless chrome=113.0.5672.126)
"
cowidev.vax.incremental.pakistan,,,2023-06-04T07:00:14,,
cowidev.vax.incremental.philippines,,,2023-06-04T07:00:14,,
cowidev.vax.incremental.poland,1.92,True,2023-06-04T07:00:16,,
cowidev.vax.incremental.qatar,,,2023-06-04T07:00:15,,
cowidev.vax.incremental.russia,23.26,True,2023-06-04T07:00:38,,
cowidev.vax.incremental.saint_lucia,,,2023-06-04T07:00:16,,
cowidev.vax.incremental.serbia,10.05,True,2023-06-04T07:00:26,,
cowidev.vax.incremental.south_africa,,,2023-06-04T07:00:17,,
cowidev.vax.incremental.spain,11.47,True,2023-06-04T07:00:28,,
cowidev.vax.incremental.sri_lanka,,,2023-06-04T07:00:17,,
cowidev.vax.incremental.suriname,15.48,True,2023-06-04T07:00:33,,
cowidev.vax.incremental.taiwan,4.64,False,2023-06-04T07:04:41,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/taiwan.py"", line 184, in main
    Taiwan().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/taiwan.py"", line 170, in export
    data = self.read().pipe(self.pipeline)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/taiwan.py"", line 38, in read
    data = self.parse_data(df, soup)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/taiwan.py"", line 122, in parse_data
    ""date"": self._parse_date(soup),
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/taiwan.py"", line 156, in _parse_date
    date_str = re.search(regex, date_raw).group(1)
AttributeError: 'NoneType' object has no attribute 'group'
",'NoneType' object has no attribute 'group'
cowidev.vax.incremental.thailand,7.18,True,2023-06-04T07:00:26,,
cowidev.vax.incremental.turkey,,,2023-06-04T07:00:20,,
cowidev.vax.incremental.united_arab_emirates,,,2023-06-04T07:00:20,,
cowidev.vax.incremental.vietnam,,,2023-06-04T07:00:20,,
cowidev.vax.incremental.who,28.91,True,2023-06-04T07:00:49,,
cowidev.vax.incremental.zambia,1.54,True,2023-06-04T07:00:22,,
